webhooks.md

```diff
@@ -0,0 +1,114 @@
+# Webhooks
+
+Whenever integration client is changing its state webhook message is sent to certain webhook url field set on that client. Webhook url is instatiated per integration and responds on action made on property level.
```

Comment:
Client is not changing state, integration is chaning state

Reply:
I would not mention the `n webhook url field set on that client.` they don't know how it is implemented.